package.json

```diff
@@ -10,7 +10,6 @@
 			"desktop",
 			"apps/*",
 			"packages/*",
-			"!packages/dataviews",
```

Comment:
We can't do this yet, it's not ready.

Reply:
It isn't ready for what? What are the risks of not registering the dataviews in the workspaces list?

Reply:
Let me remove this line